Core/include/Acts/Clusterization/InPlaceClusterization.hpp

```diff
@@ -0,0 +1,331 @@
+// This file is part of the ACTS project.
```

Comment:
This implementation does not support time-based clusterization as well. Can it be extended to cover that case?

Reply:
right now we are doing it with way: https://github.com/acts-project/acts/blob/main/Core/include/Acts/Clusterization/TimedClusterization.hpp  

do you think this is not adaptable to this implementation of yours?